src/Libraries/RevitNodes/Elements/Views/Sheet.cs

```diff
@@ -379,6 +422,27 @@ public static Sheet ByNameNumberTitleBlockAndView(string sheetName, string sheet
             return Sheet.ByNameNumberTitleBlockAndViews(sheetName, sheetNumber, titleBlockFamilyType, new[] { view });
         }
 
+        /// <summary>
+        /// Create a Revit Sheet by the sheet name, number and a title block FamilyType.
+        /// </summary>
+        /// <param name="sheetName">Sheet Name as String.</param>
+        /// <param name="sheetNumber">Sheet Number as String.</param>
+        /// <param name="titleBlockFamilyType">Titleblock that will be assigned to created Sheet.</param>
+        /// <returns>The new empty Revit sheet.</returns>
+        public static Sheet ByNameNumberTitleBlock(string sheetName, string sheetNumber, FamilyType titleBlockFamilyType)
+        {
+            if (sheetName == null)
+                throw new ArgumentNullException("sheetName");
```

Comment:
use `nameof(sheetName)` for these type of throws, easier to refactor and no magic strings.